docs/rel_0.5.0.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Version 0.5.0
+keywords: apache, hudi, release, 0.5.0, data lake, upsert,
+sidebar: mydoc_sidebar
+permalink: rel_0.5.0.html
+toc: false
+summary: "Release 0.5.0 Page"
+---
+
+Apache Hudi (incubating) Latest Stable Release is : 0.5.0-incubating
```

Comment:
backticks for `0.5.0-incubating` to highlight it

Reply:
Done.